index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    rel="stylesheet"
+  />
+</head>
+<body>
+    <header>
+        <nav>
+            <ul class="logo-part">
+                <li class="logo-link"><a href="#">My Logo</a></li>
+            </ul>
```

Comment:
- [ ] Since there are no elements listed here, you may consider wrapping this anchor tag in a div for semantic purposes.